dm/dm/master/scheduler/scheduler.go

```diff
@@ -2251,6 +2281,7 @@ func (s *Scheduler) tryBoundForSource(source string) (bool, error) {
 		}
 	}
 	if worker == nil {
+		boundSourcesNum := sourceNum
```

Comment:
Here seems that will pick a relayworker which `len(w.Bounds())` is least, Maybe we can put the log outside the loop at [here](https://github.com/pingcap/tiflow/pull/5078/files#diff-50bb6b4a0a500612d1666f0c4e4b08b0413b7bc82ae0746bc2d41d4babe93f74R2297)